### ISSUE TITLE
Finish trunk before starting branches 

### ISSUE THREAD

**Issue by MaximeHerpin** [MaximeHerpin/modular_tree#152]
It is pretty hard to control the general shape of the tree, and lower branches are generated before higher branches resulting in an often conical tree. 

To have more control over this, it would be nice to have the option to finish the trunk before allowing the branches to grow. 
This only applies when the preserve trunk property is checked. 

This will result in more cylindrical trees, which shapes will then be easily modified with the nodes. 

_Sent from my Sony F5121 using [FastHub](https://play.google.com/store/apps/details?id=com.fastaccess.github)_

**Comment by MaximeHerpin** [comment]
Done in the node workflow branch.
